recipes/reentry/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "reentry" %}
+{% set version = "1.0.2"%}
+{% set sha256 = "0fe68f19c7623d500ae0c1ca4e98d627f4d1d9832ae8bb4b00e4e471a5fdcd97"%}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/DropD/reentry/archive/v1.0.2.tar.gz
+  # PyPI version is currently missing a .json file
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  number: 0
+  entry_points:
+    - reentry = reentry.cli:reentry
+
```

Comment:
If you could delete the blank lines in the `build` section, that would be great.